cmd/serverless-init/log/log_test.go

```diff
@@ -36,6 +39,100 @@ func TestCustomWriterUnbuffered(t *testing.T) {
 	assert.Equal(t, 1, numMessages)
 }
 
+func TestCustomWriterShouldBuffer(t *testing.T) {
```

Comment:
maybe we could add a test with command === dotnet and command !== dotnet to see the diff and prevent regression?

Reply:
Done